packages/faq-extension/faq.md

```diff
@@ -47,39 +47,39 @@ Notebook. Here are a few of them you may want to try out:
 * Double click on CSV files to view them as a nicely formatted table.
 * Drag and drop notebook cells within a notebook or between notebooks.
 
-See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/latest/)
+See the [JupyterLab Documentation](http://jupyterlab.readthedocs.io/en/stable/)
```

Comment:
I'd actually move the docs link up to general so we can get people to the docs ASAP